src/ResourceTypeRegistry.js

```diff
@@ -19,8 +32,9 @@ const autoGetterSetterProps = ["dbAdapter", "beforeSave", "beforeRender",
  * JSON api type and has a number of properties defining it.
  */
 export default class ResourceTypeRegistry {
-  constructor(typeDescriptions = []) {
+  constructor(typeDescriptions = [], descriptionDefaults = {}) {
     this._resourceTypes = {};
+    this._descriptionDefaults = merge({}, globalResourceDefaults, descriptionDefaults);
```

Comment:
This merge [had been](https://github.com/ethanresnick/json-api/pull/34/files#diff-48fce73b45bb84d4bbeb7a11ce92639dR43) (I think) in the wrong order, with the globablDefaults getting merged in after the descriptionDefaults